include/measurement_kit/nettest.hpp

```diff
@@ -0,0 +1,1922 @@
+// Part of Measurement Kit <https://measurement-kit.github.io/>.
+// Measurement Kit is free software under the BSD license. See AUTHORS
+// and LICENSE for more information on the copying conditions.
+#ifndef MEASUREMENT_KIT_NETTEST_HPP
+#define MEASUREMENT_KIT_NETTEST_HPP
+
+/*-
+ *  __      __  _____ __________  _______  .___ _______    ________
+ * /  \    /  \/  _  \\______   \ \      \ |   |\      \  /  _____/
+ * \   \/\/   /  /_\  \|       _/ /   |   \|   |/   |   \/   \  ___
+ *  \        /    |    \    |   \/    |    \   /    |    \    \_\  \
+ *   \__/\  /\____|__  /____|_  /\____|__  /___\____|__  /\______  /
+ *        \/         \/       \/         \/            \/        \/
+ *
+ * Autogenerated by `./script/autoapi/autoapi`. DO NOT EDIT!!!
+ */
+
+/// \file include/measurement_kit/nettest.hpp
+///
+/// \brief C++11 API for running nettests.
+///
+/// This API is a C++11 adaptation of MK's FFI API. You should probably read
+/// about such API first. \see https://github.com/measurement-kit/measurement-kit/blob/master/include/measurement_kit/README.md.
+///
+/// Usage is as follows:
+///
+/// ### 1. Instantiate settings
```

Comment:
The following describes the steps required to use this API. I think this is nice to have, even though there is a risk that this may become outdated information if changes are not performed in sync. The lazier alternative is to refer directly to the examples folder. However, this documentation is also helpful to understand some of the decisions that I made (even though not probably all), so I'd rather keep it and risk it becomes outdated.